applications/OptimizationApplication/tests/linear_element_test/primal_parameters.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
rename the folder to sth like linear_strain_energy_test

Reply:
Will do :D

Reply:
done :)